cvmfs/directory_entry.h

```diff
@@ -18,6 +18,7 @@
 #include "bigvector.h"
 #include "compression.h"
 #include "hash.h"
+#include "libcvmfs.h"
```

Comment:
It might work fine but I'd prefer not to include this header in the internal code.